lib/lens/sender.rb

```diff
@@ -3,66 +3,40 @@
 
 module Lens
   class Sender
-    NOTICES_URI = 'api/v1/events'
-    HTTP_ERRORS = [Timeout::Error,
-                   Errno::EINVAL,
-                   Errno::ECONNRESET,
-                   EOFError,
-                   Net::HTTPBadResponse,
-                   Net::HTTPHeaderSyntaxError,
-                   Net::ProtocolError,
-                   Errno::ECONNREFUSED].freeze
+    NOTICES_URI = 'api/v1/events'.freeze
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.